src/participant/participant.ts

```diff
@@ -1692,6 +1692,54 @@ export default class ParticipantController {
     }
   }
 
+  async exportPlaygroundToLanguage({
+    code,
+    language,
+    includeDriverSyntax,
+  }: {
+    code: string;
+    language: string;
+    includeDriverSyntax: boolean;
+  }): Promise<string | null> {
+    const progressResult = await vscode.window.withProgress(
+      {
+        location: vscode.ProgressLocation.Notification,
+        title: `Exporting playground to ${language}...`,
+        cancellable: true,
+      },
+      async (progress, token): Promise<string | null> => {
+        const modelInput = await Prompts.exportToLanguage.buildMessages({
+          request: { prompt: code },
+          language,
+          includeDriverSyntax,
+        });
+        const result = await Promise.race([
+          this.streamChatResponseContentToCode({
+            modelInput,
+            token,
+            language,
+          }),
+          new Promise<null>((resolve) =>
+            token.onCancellationRequested(() => {
+              resolve(null);
+            })
+          ),
+        ]);
+
+        if (result?.includes("Sorry, I can't assist with that.")) {
+          void vscode.window.showErrorMessage(
+            "Sorry, I can't assist with that."
```

Comment:
Should we make this more descriptive? It sounds a bit generic, I'm guessing folks would expect this though if they're exporting something that generates this error. May no change needed.